### DISCUSSION TITLE
Me explican que significa porfavor

### DISCUSSION THREAD

**Discussion by CarlitosRa** [community/community#130075] [Enterprise]
### Body

![Screenshot_20240627-165652](https://github.com/community/community/assets/173897953/ed82f70d-09bc-45c7-b489-5f3c119398b1)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by davevad93** [comment]
Hi @CarlitosRa ,


It's an explanation on how to import a repository from `Mercurial` (which is another source code management tool) into `GitHub`.
Previously this could be done directly with `GitHub Importer` but as of April 12, 2024, `GitHub` no longer allows to import `Mercurial`, `Subversion` and `Team Foundation Version Control (TFVC)` repositories, for more information take a look at this:

- [Deprecation: Importing non-Git repositories with GitHub Importer](https://github.blog/changelog/2023-04-17-deprecation-importing-non-git-repositories-with-github-importer/)

**Comment by queenofcorgis** [comment]
Hi @CarlitosRa , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
